test/unit/auth/tenant.spec.ts

```diff
@@ -361,6 +371,14 @@ describe('Tenant', () => {
         }).to.throw('"RecaptchaConfig.managedRules" must be an array of valid "RecaptchaManagedRule".');
       });
 
+      it('should throw on non-boolean useAccountDefender attribute', () => {
+        const tenantOptionsClientRequest = deepCopy(clientRequestWithRecaptcha) as any;
+        tenantOptionsClientRequest.recaptchaConfig.useAccountDefender = 'yes';
```

Comment:
nit: Please see the comment in `test/unit/auth/project-config.spec.ts`

Reply:
Done